crnk-core/src/main/java/io/crnk/core/engine/internal/document/mapper/ResourceMapper.java

```diff
@@ -123,7 +126,18 @@ protected void setAttributes(Resource resource, Object entity, ResourceInformati
 					setAttribute(resource, field, value);
 				}
 			}
+
+		}
+
+		if (resourceInformation.getAnyFieldAccessor() != null) {
+			AnyResourceFieldAccessor anyFieldAccessor = resourceInformation.getAnyFieldAccessor();
+			Object o = anyFieldAccessor.getValues(entity);
```

Comment:
The anyFieldAccessor could return a Map directly I guess?

Reply:
OK I will update it, sound good